recyclerview/src/main/java/me/henrytao/recyclerview/BaseAdapter.java

```diff
@@ -274,15 +296,31 @@ public void onItemRangeRemoved(int positionStart, int itemCount) {
           }
         }
       };
-      mBaseAdapter.registerAdapterDataObserver(mAdapterDataObserver);
+      for (RecyclerView.Adapter adapter : mBaseAdapters) {
+        adapter.registerAdapterDataObserver(mAdapterDataObserver);
+      }
     }
     if (notifyDataSetChanged) {
       notifyDataSetChanged();
     }
   }
 
+  private int[] getBaseAdapterInfo(int dataPosition) {
```

Comment:
I don't think so. This method is used internally. It just let us know the index of BaseAdapter and the dataPosition of it.